package-lock.json

```diff
@@ -522,6 +522,37 @@
         }
```

Comment:
Travis is failing because this file is outdated, could you regenerate it once you remove the `^` from the new dependencies added?

https://travis-ci.org/brunocodutra/webapp-webpack-plugin/jobs/383896536#L490